src/components/Atoms/EditButton/index.jsx

```diff
@@ -0,0 +1,42 @@
+import styled from "styled-components";
+import React from "react";
+import pencil from "../../../assets/svg/pencil.svg";
+import COLOR from "../../../variables/color";
+
+export const EditButton = (props) => {
+    return (
+        <StyledButton onClick={props.onClick}>
+            <Icon/>
+        </StyledButton>
+    );
+}
+
+const Icon = () => {
+    return <img src={pencil}/>;
+}
+
+const StyledButton = styled.button`
+    position: relative;
+    font-size: 20px;
+    color: ${COLOR.LIGHT_GRAY};
+    background: none;
+    border: none;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border-radius: 50%;
+    overflow: hidden;
+
+    &::after {
+        content: "";
+        position: absolute;
+        width: 24px;
+        height: 24px;
+        background: ${COLOR.LIGHT_GRAY_TRANSPARENT};
+        border-radius: 50%;
+    }
+
+    &:hover::after {
+        background: ${COLOR.LIGHT_GRAY_TRANSLUCENT};
+    }
```

Comment:
多分サイト制作課題の時の延長線上で、色をオーバーレイするためにafterを使ってくれているのかと思うんだけど、今回の場合ホバーなので、ボタンをホバーしたらボタンにバックグラウンドカラーを適用するで対応可能です！

```suggestion
    &:hover::after {
        background: ${COLOR.LIGHT_GRAY_TRANSLUCENT};
        width: 24px;
        height: 24px;
        border-radius: 50%;
    }
```